wordpress-mobile/AztecEditor-iOS
# Implements a Color provider

Fixes #https://github.com/wordpress-mobile/gutenberg-mobile/issues/1603#issuecomment-559841868

This PR adds support for the standard HTML colors and also provides a way for provide default colors for formatters.

<img width="365" alt="Screenshot 2019-12-02 at 16 45 09" src="https://user-images.githubusercontent.com/651601/69977701-2e1e5580-1523-11ea-8466-b11299349bfd.png">

To test:
 - Start the demo app
 - Check if the colors are correctly set on the Colors line
 - Check if the code tag has the correct background color.
